_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,269 @@
+
+Interesting Issues:
+
+- [Using browserslist](https://github.com/babel/babel-preset-env/issues/26)
+- The next big feature is the dynamic checking of polyfills as described in [babel/babel-preset-env#20](https://github.com/babel/babel-preset-env/issues/20) and PR in [babel/babel-preset-env#56](https://github.com/babel/babel-preset-env/pull/56).
+
+### Linting via [babel-eslint](https://github.com/babel/babel-eslint)
+
+ESLint doesn't support new language features until they reach Stage 4 of the proposals process. For this reason we maintain [babel-eslint](https://github.com/babel/babel-eslint), a custom ESLint parser so you can continue to lint JavaScript with experimental syntax.
+
+I think this project was definetely one of the hardest projects to work on: because it is literally a compatibility layer between Babel and ESLint there is a constant need to update when either project updated and a high risk of breaking changes due to how it worked (monkey-patching). It was unforunate to get issues like [babel/babel-eslint#243](https://github.com/babel/babel-eslint/issues/243) or [babel/babel-eslint#267](https://github.com/babel/babel-eslint/issues/267).
+
+To that end, we'd like to lessen the maintenance burden of this project by improving our scope and traversal interop with ESLint. It might even be interesting to be able to write ESLint rules using babel APIs or vice versa?
+
+Interesting Issues:
+
+- Is it ironic that [babel/babel-eslint#88](https://github.com/babel/babel-eslint/issues/88) is just as relevant now?
+- [Running smoke tests on ESLint](https://github.com/babel/babel-eslint/issues/62)
+
+### Minification
+
+[Babili](https://github.com/babel/babili) is our new Babel-powered minifier, enabling you to produce minified code while targeting the latest browsers.
+
+Check out our [blog post](http://babeljs.io/blog/2016/08/30/babili) for more info.
+
+Interesting Issues:
+
+TODO
+
+### Codemods / Refactoring
+
+Other relevant projects: [jscodeshift](https://github.com/facebook/jscodeshift), [recast](https://github.com/benjamn/recast), [Lebab](https://github.com/lebab/lebab).
+
+By passing in [Recast](https://github.com/benjamn/recast) in the options, Babel can also be run on source code and can power the future of codemods. This feature was just made possible so we're looking forward to making it easier use and seeing the transformations it can enable.
+
+Interesting Issues:
+
+- [Rewriting Lebab as a Babel transformer](https://github.com/lebab/lebab/issues/138)
+- [Babel integration with jscodeshift](https://github.com/facebook/jscodeshift/issues/168)
+
+Check out the [6.16.0 blog post](https://babeljs.io/blog/2016/09/28/6.16.0#new-feature) for more info!
+
+### Code Coverage / Instrumentation
+
+With tools like [nyc](https://github.com/istanbuljs/nyc) and [babel-plugin-instanbul](https://github.com/istanbuljs/babel-plugin-istanbul).
```

Comment:
Perhaps mention that these tools use babel's modular API to add code coverage to your codebase, regardless of plugins that you're using (ES2015, JSX/Flow, etc).

Interesting Issues:

* work and progress discussion about how the heck we'll eventually get the nyc and Istanbul projects completely merged: https://github.com/istanbuljs/nyc/issues/309
* there are still edge-cases where we fail at instrumenting transpiled code https://github.com/istanbuljs/babel-plugin-istanbul/issues/63

In general, would love help on both these projects and am willing to mentor new OSS maintainers.